powersimdata/data_access/csv_store.py

```diff
@@ -50,15 +50,21 @@ def get_table(self):
         :return: (*pandas.DataFrame*) -- the specified table as a data frame.
         """
         filename = self._FILE_NAME
-        try:
-            return self._get_table(filename)
-        except:  # noqa
-            return self._get_table(filename + ".2")
+        orig = self._get_table(filename)
+        blob = self._get_table(filename + ".2")
+        df = pd.concat([orig, blob])
+        return df[~df.index.duplicated()]
 
     def _get_table(self, filename):
-        self.data_access.copy_from(filename)
-        with self.data_access.get(filename) as (f, _):
-            return _parse_csv(f)
+        try:
+            self.data_access.copy_from(filename)
+        except:  # noqa
+            pass
+        try:
+            with self.data_access.get(filename) as (f, _):
+                return _parse_csv(f)
+        except:  # noqa
+            return pd.DataFrame()
```

Comment:
Reading through the logic here, it seems it will also work if one doesn't have a local copy and not connects to the internet, but have a local copy of `csv.2`, i.e. the one on the blob storage? It will be picked up by the second `try` block in the `blob` query in l.54. It should be fine though, just wanna make sure that's the intended behavior.

Reply:
yep, that's more or less the intent. I wasn't specifically considering lack of internet, since e.g. if there are no local copies you'd get an empty data frame, and I'm not what downstream failures would look like. But say you had a local `.csv.2` as well as local scenario files (pf, pg, etc), then I think everything would still work in that case.